lib/account/account-manager.js

```diff
@@ -1,36 +1,80 @@
 "use strict";
-/**
- * Manages the business logic of account controls.
- */
 
 /**
- * The object sent in which has information needed to validate the user
+ * The object sent container information needed to validate the user
```

Comment:
I don't get this.  The object sent container information?  What is container information?